python/tvm/relax/transform/legalize_ops.py

```diff
@@ -826,15 +834,17 @@ def __init__(self, mod: IRModule, customize_legalize_map: Dict[str, LegalizeFunc
 
             def _convert_op(self, call: Call) -> Expr:
                 if call.op.name in self.legalize_map:
-                    # We only transform the op calls with known shape values
-                    if not all(
-                        [has_known_shape_value(arg.struct_info) for arg in call.args]
-                    ) or not has_known_shape_value(call.struct_info):
+                    try:
+                        return self.legalize_map[call.op.name](self.builder_, call)
+                    except Exception as e:  # pylint: disable=broad-except
```

Comment:
I think it is good to have clear requirement on shape here, as this is the prerequisite for a CallNode to be legalized. We can have the try-catch here to capture other potential exceptions, but it is good to keep the shape check here.